### PR TITLE
setup pre commit

### PR DESCRIPTION
**Pull request by gurdeep111994**

### CODE REVIEW

**Comment by gurdeep111994** [comment]
@chirgjn need to add commands for linting and test cases. we didn't have scripts in package.json for this branch, once setup nest PR merged I'll add both commands

File: package.json [hunk 1]
```diff
@@ -41,6 +42,7 @@
     "eslint": "^7.30.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^3.4.0",
+    "husky": "*",
```
Comment (chirgjn) [inline package.json]: Please specify a version here

**Review by chirgjn — CHANGES_REQUESTED** [review]
This PR contains a lot of unrelated changes.

Why are we changing these single quotes to double quotes?

File: package.json (hunk 1 above)
Comment (gurdeep111994) [reply package.json]: eslint not allowing a single quotes to commit

Comment (chirgjn) [reply package.json]: Please try the format script

**Review by chirgjn — CHANGES_REQUESTED** [review]

**Review by chirgjn — APPROVED** [review]
LGTM

**Pull request merged**